# Background
이전 연구에서 3양봉의 수익률이 더 높다는 사실을 알게 되었다.  
그러나 3양봉이라 하더라도 반드시 성공하는 것은 아니며 조심해야 될 매수등락률이 있을 것이다.  
유튜브: https://youtu.be/ULDNdiwZQfk

# Methods

In [1]:
import pandas as pd

In [17]:
df = pd.read_csv('data.csv')
df['매수등락률'] = df['매수등락률'].apply(lambda x : int(x))
grouped = df.groupby(['양봉개수', '매수등락률'])['수익률'].mean()
grouped

양봉개수  매수등락률
0     7       -3.530
      8        2.140
      9        0.270
      10       0.345
      11      -1.520
               ...  
6     25      -2.430
      26       1.620
9     11      -2.020
      20       3.300
10    15      -0.300
Name: 수익률, Length: 78, dtype: float64

In [19]:
grouped[3]

매수등락률
6    -1.310000
9     2.100000
10   -0.870000
11   -0.993333
13    3.660000
14   -0.300000
15    1.960000
16   -2.000000
21    1.903333
24   -1.470000
27    6.680000
Name: 수익률, dtype: float64

# Results
3양봉은 매수등락률 27%일 때 수익률이 압도적으로 높다.

# Conclusion
3양봉은 상따자리에서 큰 시너지가 난다.